spec/rspec/matchers/built_in/compound_spec.rb

```diff
@@ -262,13 +262,19 @@ def expect_block
 
     describe "expect(...).to matcher.and(other_matcher)" do
 
-      it_behaves_like "an RSpec matcher", :valid_value => 3, :invalid_value => 4, :disallows_negation => true do
+      it_behaves_like "an RSpec value matcher" do
```

Comment:
Now that value and blocks specs are separate I think that for "an RSpec value matcher" it's possible to keep passing the arguments via shared example, not using `let`'s.
It will also reduce the size of this pull request.

Reply:
+512 -> +444, less churn.